airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/DatabaseConfigPersistenceLoadDataTest.java

```diff
@@ -0,0 +1,162 @@
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.config.persistence;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.Lists;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.ConfigSchema;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSync;
+import io.airbyte.db.instance.configs.ConfigsDatabaseInstance;
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Stream;
+import org.jooq.DSLContext;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+/**
+ * Unit test for the {@link DatabaseConfigPersistence#loadData} method.
+ */
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class DatabaseConfigPersistenceLoadDataTest extends BaseDatabaseConfigPersistenceTest {
+
+  private final ConfigPersistence seedPersistence = mock(ConfigPersistence.class);
+
+  @BeforeAll
+  public static void setup() throws Exception {
+    database = new ConfigsDatabaseInstance(container.getUsername(), container.getPassword(), container.getJdbcUrl()).getAndInitialize();
+    configPersistence = spy(new DatabaseConfigPersistence(database));
+    database.query(ctx -> ctx.execute("TRUNCATE TABLE airbyte_configs"));
+  }
+
+  @AfterAll
+  public static void tearDown() throws Exception {
+    database.close();
+  }
+
+  @BeforeEach
+  public void resetPersistence() {
+    reset(seedPersistence);
+    reset(configPersistence);
+  }
+
+  /**
+   * When the database is empty, seed will be copied into the database.
```

Comment:
nit: we can use the `DisplayName` annotation for this - one good side effect is this will display in the output when running the test.

Reply:
👍🏻 for test order and comments

Reply:
Good point. Will do.